core/src/main/java/com/wttech/aem/acm/core/osgi/InstanceInfo.java

```diff
@@ -27,4 +40,22 @@ public boolean isRunMode(String runMode) {
     public Set<String> getRunModes() {
         return slingSettings.getRunModes();
     }
+
+    public InstanceSettings getInstanceSettings() {
+        boolean publish = isPublish();
+        String propUrl = bundleContext.getProperty("PRODUCTINFO_VERSION");
```

Comment:
I would assume to check more than this single property;
I feel like some one them will disappear; I hope not this one

Reply:
`boolean cloud = isCloud()` ; do not split retrieval and computation; the one method for them all makes more sense